packages/kit/src/runtime/client/renderer.js

```diff
@@ -275,43 +275,49 @@ export class Renderer {
 			this._init(navigation_result);
 		}
 
-		const { hash, scroll, keepfocus } = opts || {};
+		// opts must be passed if we're navigating
+		// they will not be supplied if we're simply invalidating
+		if (!opts) {
+			await 0;
+		} else {
+			const { hash, scroll, keepfocus } = opts || {};
```

Comment:
```suggestion
			const { hash, scroll, keepfocus } = opts;
```
nit: We can remove the opts fallback